lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -229,6 +229,12 @@
             - "I(True) enable menu to select boot device, I(False) to disable it. By default is chosen by oVirt/RHV engine."
         type: bool
         version_added: "2.5"
+    bios_type:
+        description:
+            - "Sets bios type, necessary for some operating systems and secure boot.
+        choices: [ i440fx_sea_bios, q35_ovmf, q35_sea, q35_secure_boot ]
+        default: i440fx_sea_bios
```

Comment:
from 4.4 there is `cluster_default` maybe remove the default option and keep it None

Reply:
done, keeps the omitted behavior the same.